lib/model.js

```diff
@@ -2221,7 +2221,7 @@ class Model {
    * The successful result of the promise will be (instance, built)
    *
    * @param {Object}   options find options
-   * @param {Object}   options.where A hash of search attributes.
+   * @param {Object}   options.where A hash of search attributes. Will be appended to defaults if create runs.
```

Comment:
If `where` is a plain object it will be appended with `defaults` to build a new instance